### ISSUE TITLE
Add a note that publishers created the accessibility metadata that reflects the EPUB they released into many distribution channels

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#95]
This note should be before the heading "Before you start writing"

a line that declares that metadata should be written (and consumed) as coming from the publisher but some values may not be accurate depending on the reading system that is used?

The  Note that accessibility metadata is from publishers  that gets distributed widely should include that the ingestion process has the potential to introduce changes. Distributors should make it clear to end users what they are doing to the ingested EPUB.

**Comment by GeorgeKerscher** [comment]
It is merged.

 

Best

George

**Comment by RachelComerford** [comment]
Yes. It could also say that the titles contains MathML but the support of mathml varies from one reader to another. This could apply to many accessibility features.
